src/tests/ftest/harness/coverage.yaml

```diff
@@ -0,0 +1,38 @@
+hosts:
+  test_servers: 1
+  test_clients: 1
+
+timeout: 270
+
+server_config:
+  name: daos_server
+  engines_per_host: 1
+  engines:
+    0:
+      targets: 4
+      nr_xs_helpers: 0
+      storage:
+        0:
+          class: ram
+          scm_mount: /mnt/daos
+
+pool:
+  size: 2G
+
+container:
+  control_method: daos
+  type: POSIX
+
+ior: &ior_base
+  ppn: 4
+  api: DFS
+  transfer_size: 64K
+  block_size: 128K
```

Comment:
This could potentially be super slow due to the VM slowness issue. I'd recommend reducing drastically to like 1K or less, but even that could be slow